docs/docs/workflows/iam.md

```diff
@@ -0,0 +1,210 @@
+
+You can also run `constellation config instance-types` to get the list of all supported options.
+
+</tabItem>
+<tabItem value="gcp" label="GCP">
+
+You must be authenticated with the [GCP CLI](https://cloud.google.com/sdk/gcloud) in the shell session.
+
+```bash
+constellation iam create gcp --projectID=yourproject-12345 --zone=europe-west1-a --serviceAccountID=constell-test
+```
+
+:::info
+This command creates IAM configuration in the GCP project `yourproject-12345` on the GCP zone `europe-west1-a` using a new service account `constell-test`.
+:::
+
+Paste the output into the corresponding fields of the `constellation-conf.yaml` file created by the `constellation config generate` command.
+
+By default, Constellation uses `n2d-standard-4` VMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. Supported are all machines from the N2D family. Refer to [N2D machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines) or run `constellation config instance-types` to get the list of all supported options.
+
+</tabItem>
+<tabItem value="aws" label="AWS">
+
+You must be authenticated with the [AWS CLI](https://aws.amazon.com/en/cli/) in the shell session.
+
+```bash
+constellation iam create aws --zone=eu-central-1a --prefix=constellTest
+```
+
+:::info
+This command creates IAM configuration for the AWS zone `eu-central-1a` using the prefix `constellTest` for all named resources being created.
+:::
+
+Paste the output into the corresponding fields of the `constellation-conf.yaml` file created by the `constellation config generate` command.
+
+Constellation OS images are currently replicated to the following regions:
+    * `eu-central-1`
+    * `us-east-2`
+    * `ap-south-1`
+
+If you require the OS image to be available in another region, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md&title=Support+new+AWS+image+region:+xx-xxxx-x).
+
+You can find a list of all [regions in AWS's documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
+
+By default, Constellation uses `m6a.xlarge` VMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. Supported are all nitroTPM-enabled machines. Refer to the [list of nitroTPM-enabled instance types](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enable-nitrotpm-prerequisites.html) or run `constellation config instance-types` to get the list of all supported options.
+
+</tabItem>
+</tabs>
+
+Alternatively, you can manually create the IAM configuration on your CSP.
```

Comment:
We could also merge the manual boxes with the automated boxes to achieve a better "flow" in the docs. Now I need to notice that this is not interesting for me, and skip some amount of content. 

Another option would be to have something like: "If you created these resources automatically continue with <link to create cluster>" as we have further below.

Reply:
I also thought about a "drawer" which you could open up and close, but didn't find a suitable option in Docusaurus. I think the manual creation boxes might be off-putting to a user because of their size. Ideally I would "hide" them or give the user a big hint to skip them if they used the command (which is recommended either way).

Reply:
That seems perfect! Don't know why i didn't find it before